src/main/resources/messages_en.properties

```diff
@@ -651,6 +651,17 @@ program.favorite.type.not.identified=The following favorite type {0} is not defi
 #Inventory Module
 location.required=Location is required
 location.invalid=Invalid location identifier
+location.is.used.in.lots=The location is used in lots
```

Comment:
I think we can improve the error messages in most cases:
Location Id {0} can not be deleted because is being used in Lots